build/targets/standard-lifecycle.targets

```diff
@@ -1,17 +1,44 @@
 <Project>
-  <Target Name="Default" DependsOnTargets="Build" />
 
-  <Target Name="Build" DependsOnTargets="Clean;Initialize;Compile;Test;Verify;Publish" />
+  <PropertyGroup>
+    <Configuration>Debug</Configuration>
+  </PropertyGroup>
 
-  <Target Name="Clean" />
+  <ItemGroup>
+    <SolutionsToBuild Include="$(RepositoryRoot)\*.sln" />
```

Comment:
The goal here is to make the default case work, which is that we have one sln per repo. MVC and any other solution that has multiple solutions would put something like this in to build/repo.targets:
```xml
<SolutionsToBuild Remove="Mvc.NoFun.sln" />
```

We don't have a good way to determine "RepoName" :-/ as the folder name containing the source code is not guaranteed to match the sln name.